.ci/sonar.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+#set checkstyle sonar profile
+curl -X POST -u admin:admin -F 'backup=@config/default_sonar_profile.xml' -v http://localhost:9000/api/profiles/restore
+
+#execute inspection
+mvn sonar:sonar -Dsonar.language=java -Dsonar.profile=checksyle-profile
+
+# Uncomment following to get HTML report.
+# mvn sonar:sonar -Dsonar.analysis.mode=preview -Dsonar.issuesReport.html.enable=true \
+#        -Dsonar.language=java -Dsonar.profile=checksyle-profile
+
+
+#get and parse response from sonar
+#give some time to sonar for report processing
+sleep "60"
+curl -u admin:admin -v http://localhost:9000/api/issues/search?componentRoots=com.puppycrawl.tools:checkstyle \
+        > response.json
+
+
+REPORT="$(cat response.json)"
+OUTPUT="$(cat response.json | jq '.total')"
+rm response.json
+
+#print number of found issues
+if test -n "$OUTPUT"; then
+    echo "$REPORT" | jq
```

Comment:
Just do cat of result file and redirect it to jq

Reply:
done